lib/addons/jupyterhub/index.ts

```diff
@@ -108,7 +108,6 @@ export class JupyterHubAddOn extends HelmAddOn {
         this.options = this.props as JupyterHubAddOnProps;
     }
 
-    @dependable(AwsLoadBalancerControllerAddOn.name)
     deploy(clusterInfo: ClusterInfo): Promise<Construct> {
```

Comment:
Please address the github actions warning: dependable is defined but never used.

Reply:
see above.